### DISCUSSION TITLE
GLIBC_2.39 not found on new project start ubuntu 22.04

### DISCUSSION THREAD

**Discussion by einarpersson** [tauri-apps/tauri#12037] [General]
Hi,

I just wanted to try tauri and ran 
```
sh <(curl https://create.tauri.app/sh)
```
as recommended on linux (Ubuntu 22.04 under WSL2)

However, I just got a error
```
/tmp/tmp.LHg9AyONYv/create-tauri-app: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.39' not found (required by /tmp/tmp.LHg9AyONYv/create-tauri-app)
```

without any instructions on where to go from here. There are no notes anticipating this error in the quickstart guide either, which I suppose will be what a lot of people encounter on their first try.

I guess I have one question and one commment:

1. Where to go from here?
2. If this is an expected error for a lot of people, then perhaps there should be a note in the quickstart and/or as output from the installation script

**Comment by FabianLars** [comment]
Can you open an issue here? https://github.com/tauri-apps/create-tauri-app It's not supposed to use github's `ubuntu-latest` runner to build the binaries.

As a workaround for now you can use
```
cargo install create-tauri-app --locked
cargo create-tauri-app
```
